Formula/libomxil-bellagio.rb

```diff
@@ -0,0 +1,24 @@
+class LibomxilBellagio < Formula
+  desc "Open-source implementation of the OpenMAX Integration Layer"
+  homepage "https://omxil.sourceforge.io/"
+  url "https://downloads.sourceforge.net/project/omxil/omxil/Bellagio%200.9.3/libomxil-bellagio-0.9.3.tar.gz"
+  sha256 "593c0729c8ef8c1467b3bfefcf355ec19a46dd92e31bfc280e17d96b0934d74c"
+  license "LGPL-2.1-or-later"
+
+  depends_on "pkg-config" => [:build, :test]
+  depends_on :linux
+
+  def install
+    ENV.deparallelize
```

Comment:
Would be good to report this upstream if it hasn't been already.

Reply:
This project is really old—the latest commit was in 2011

Reply:
Can we try a build without ENV.deparallelize?

Reply:
I get this error:

```
/usr/bin/ld: cannot find -lomxil-bellagio
collect2: error: ld returned 1 exit status
```

Reply:
Interested in what @SMillerDev thinks here but this is when (the latest commit was in 2011, doesn't build perfectly) we'd consider rejecting this into Homebrew/core. Depends what depends on this, really, don't want to block it if it's going to block other stuff.

Reply:
This is used only by `mesa` on Linux for supporting OpenMAX IL, but we probably could get away with removing it since OMX seems to be pretty obscure anyways. Wikipedia page: https://en.wikipedia.org/wiki/OpenMAX

Reply:
Yeh, good call 👍🏻

Reply:
Agreed that if we can disable that feature in mesa, we do not need to add `libomxil-bellagio` at all.

Reply:
Yeah, agree with Mike although I'd push for upstreaming fixes regardless of last commit etc. Not supporting something antiquated is a fine option too.